extpreimage/README.md

```diff
@@ -0,0 +1,68 @@
+extpreimage
+=====
+
+This extpreimage package implements a client for a generic External Preimage
+Service running a gRPC server.
+
+## Generate protobuf definitions
```

Comment:
Could probably remove this and add it to our lnd wiki or something since it doesn't look related to the service itself

Reply:
In order to use protobuf in golang, you need to generate files (with the suffix `.pb.go`) from the `.proto` file. This provides instruction for how to do that.